packages/std-client/package.json

```diff
@@ -50,43 +45,24 @@
     "@latticexyz/store": "workspace:*",
     "@latticexyz/utils": "workspace:*",
     "@latticexyz/world": "workspace:*",
+    "abitype": "^0.8.2",
+    "ethers": "^5.7.2",
+    "mobx": "^6.7.0",
+    "react": "^18.2.0",
+    "rxjs": "^7.5.5",
+    "viem": "^0.3.18"
+  },
+  "devDependencies": {
     "@types/jest": "^27.4.1",
     "@types/lodash": "^4.14.182",
     "@types/react": "^18.2.6",
     "@types/react-collapse": "^5.0.1",
     "@types/uuid": "^8.3.4",
-    "ethers": "^5.7.2",
-    "lodash": "^4.17.21",
-    "mobx": "^6.7.0",
-    "react": "^18.2.0",
-    "react-collapse": "^5.1.1",
-    "react-dom": "^18.2.0",
-    "rimraf": "^3.0.2",
-    "rxjs": "^7.5.5",
-    "tslib": "^2.5.0",
     "tsup": "^6.7.0",
     "tsx": "^3.12.6",
     "typedoc": "0.23.21",
     "typedoc-plugin-markdown": "^3.13.6",
-    "typescript": "^4.9.5",
```

Comment:
do we not need typescript as a dev dependency? (it seems to work without but i'm surprised)

Reply:
It's installed at the root since ~all packages need it and need to be aligned to the same version.